geopy/geocoders/osm.py

```diff
@@ -443,6 +444,8 @@ def reverse(
 
             .. versionadded:: 1.14.0
 
+        :param str zoom: Level of detail required for the address.
```

Comment:
If my understanding is correct, `zoom` would usually be a float or an integer rather than a string.

Maybe the type should be documented as union?

```
:param str zoom: Level of detail required for the address.
:type zoom: str, int, or float.
```

Reply:
Done.